spec/source_spec.rb

```diff
@@ -48,21 +49,41 @@
   end
 
   context "#load_credentials" do
-    it "calls api 3 times" do
-      endpoint, authentication, credentials = double, double, double
-      allow(endpoint).to receive(:[])
-      allow(authentication).to receive(:[])
+    context "when there is a receptor node" do
+      it "still computes the digest even without auth/creds" do
+        endpoint = {"receptor_node" => "asdf"}
+        api = double("api")
+
+        allow(api).to receive_messages(:get_endpoint       => endpoint,
+                                       :get_authentication => nil,
```

Comment:
@lindgrenj6 do you want to set the get_authentication to return 'receptor_node'? Since the authtype is set to receptor_node.

Reply:
I mean I could - but we dont do anything with that data. The _only_ piece we care about is the `receptor_node` field on the `Endpoint`, everything else is optional (as far as orchestrator is concerned). As long as we have the `receptor_node`, we can spin up the collector. 

@syncrou maybe I'm missing something here - we don't use the authType anywhere right? I'm not aware that we do.